server/chat-commands/info.ts

```diff
@@ -582,7 +582,10 @@ export const commands: ChatCommands = {
 						Height: `${pokemon.heightm} m`,
 					};
 					details["Weight"] = `${pokemon.weighthg / 10} kg <em>(${weighthit} BP)</em>`;
-					if (pokemon.canGigantamax) details["G-Max Move"] = pokemon.canGigantamax;
+					if (pokemon.forme === 'Gmax') {
+						const gmaxMove = dex.getSpecies(pokemon.baseSpecies).canGigantamax;
+						if (gmaxMove) details["G-Max Move"] = gmaxMove;
+					}
```

Comment:
I think they should still display on the base form as well, and this also doesn't work for formes like Urshifu-Rapid-Strike-Gmax.

Reply:
```ts
if (pokemon.forme === 'Gmax') {
	const gmaxMove = dex.getSpecies(pokemon.name.replace('-Gmax', '')).canGigantamax;
	if (gmaxMove) details["G-Max Move"] = gmaxMove;
}``` This is sort of hacky (but does what you want); is there a way to programmatically get the specific form a Gmax mon transformed from?

Reply:
I don't see why they should be on the base.

``changesFrom`` is what you're looking for.

Reply:
I thought `changesFrom` was what I wanted originally, but it turns out it's undefined for Gigantamaxes. (I originally used `baseSpecies` as per this comment https://github.com/smogon/pokemon-showdown/blob/master/sim/dex-data.ts#L699.)

Reply:
`baseSpecies` should basically never be used. The linked comment is outdated; I believe `changesFrom` now works correctly for megas/primals.

Reply:
It doesn't work for Gigantamax formes, though. 
<img width="354" alt="image" src="https://user-images.githubusercontent.com/56906084/89134703-f580b280-d4db-11ea-9d35-fd7d0abf6942.png">

Reply:
We might want to fix that.